theano/gof/fg.py

```diff
@@ -739,11 +739,11 @@ def __repr__(self):
         return self.__str__()
 
     # clone #
-    def clone(self, check_integrity=True):
+    def clone(self, check_integrity=True, attach_feature=False):
```

Comment:
This changes the default behavior for all clients, which is a bad idea.  Make it default to True and call it with False when you need it.

Also I wonder why you need this change.  You don't seem to clone FunctionGraphs anywhere.